testdata/generate_test_data/create_complex_circular_references.py

```diff
@@ -67,13 +71,15 @@ def _make_xml_text_prop(target_res: etree._Element | list[etree._Element]) -> et
     return excel2xml.make_text_prop(name=":hasRichtext", value=excel2xml.PropertyElement(salsah_link, encoding="xml"))
 
 
-def _make_resptr_prop(target_res: list[etree._Element] | etree._Element) -> etree._Element:
+def _make_resptr_prop(
+    target_res: list[etree._Element] | etree._Element, property_name: str = ":hasResource1"
```

Comment:
my personal preference is to put multiple parameters on multiple lines

Reply:
I do like that too, if there are many or the type hints are extremely long and complex. But in this case the types are quite repetitive and the functions very short. So I do not want to end up with a function that has a longer parameter and output declaration than code. So I decided against that.